web-api/src/personalization/routes.rs

```diff
@@ -684,6 +685,9 @@ async fn semantic_search(
     .await?;
 
     if let Some(personalize) = personalize {
+        if matches!(strategy, SearchStrategy::Knn) {
+            normalize_knn_scores(&mut documents);
+        }
```

Comment:
:wrench:  The documents already have been normalized  with `rescale_knn_scores` in `knn_search` called by `get_by_embedding`.